test/sql/copy/csv/test_union_by_name.test

```diff
@@ -63,8 +63,6 @@ BIGINT	BIGINT	BIGINT
 # Test timestamp type
 statement error
 SELECT * FROM read_csv_auto(['data/csv/union-by-name/ubn1.csv', 'data/csv/union-by-name/ubn2.csv', 'data/csv/union-by-name/ubn3.csv', 'data/csv/union-by-name/ubn4.csv'])
-----
-Could not convert string
```

Comment:
Does this error message need to be removed, or can it just be changed to something else?